Makefile

```
@@ -11,13 +11,15 @@ PLATFORMS ?= linux_amd64
 # ====================================================================================
 # Setup Kubernetes tools
 
-UP_VERSION = v0.14.0
+UP_VERSION = v0.21.0
 UP_CHANNEL = stable
-UPTEST_VERSION = v0.2.1
+UPTEST_VERSION = v0.9.0
```

(nit) we can use latest right away
```suggestion
UPTEST_VERSION = v0.10.0
```

---

Will merge all renovate after this PR